lib/docs/.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules
+.DS_Store
+dist
+dist-ssr
+*.local
+storybook-static
```

Comment:
I think adding new line at the end of file is a good practice.

Reply:
ah yes, me too, I think we can address it using prettier or something similar in another PR?

Reply:
Yeah, next PR will be the prettier + CI linter